governance/votes/Executive Vote - June 16, 2023.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Template - [Executive Vote] Out-of-Schedule Executive Vote - RWA014-A (Coinbase Custody) DAO Resolution - June 16, 2023
+summary: Approving a DAO Resolution pertaining to RWA014-A (Coinbase Custody).
```

Comment:
Summary might as well say what it's intended to achieve? 'To help ensure sufficient DAI liquidity' or something.

Reply:
updated